catalogs/catalog-hadoop/src/main/java/org/apache/gravitino/catalog/hadoop/HadoopCatalogOperations.java

```diff
@@ -132,6 +132,20 @@ public void initialize(
                     Map.Entry::getValue));
     bypassConfigs.forEach(hadoopConf::set);
```

Comment:
The initialization of Hadoop conf here seems to be useless. It's better to move these codes above into the `DefaultConfigurationProvider`.

Reply:
Yeah, you are right, I will do it according to your suggestion.